ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigsTablePage.tsx

```diff
@@ -397,3 +424,6 @@ function ScanConfigsTablePage({
 }
 
 export default ScanConfigsTablePage;
+function isFeatureFlagEnabled(arg0: string) {
+    throw new Error('Function not implemented.');
+}
```

Comment:
😅  maybe throw in a comment here as a TODO to delete once implemented so it doesn't get lost?